src/Cake.Web/App_Data/blog/2016-09-16-cake-v0.16.0-released.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Cake v0.16.0 released
+category: Release Notes
+author: patriksvensson
+---
+
+Version 0.16.0 of Cake has been released, and it's a big release!
+
+Not only are we supporting the full .NET framework (*v4.5*), but from this version
+we will also support running Cake on platforms capable of running .NET Core (*netstandard1.6*).
+
+For this to work, you will have to install the `Cake.CoreCLR` package instead
+of the normal `Cake` package. A blog post describing this more in detail 
+will be published soon.
+
+Please let us know if you run into any issues with this latest release.
```

Comment:
@patriksvensson I think it would be good to include the normal list of contributors to this release as well.

The output from my "tool" is the following:

```
- [gep13](https://github.com/gep13)
- [devlead](https://github.com/devlead)
- [patriksvensson](https://github.com/patriksvensson)
- [dbertram](https://github.com/dbertram)
- [cpx](https://github.com/cpx)
```

I think we should add this into the blog post